webapi/src/main/scala/org/knora/webapi/responders/v2/ResourcesResponderV2.scala

```diff
@@ -2000,11 +2003,11 @@ final case class ResourcesResponderV2(
 }
 
 object ResourcesResponderV2 {
-  val layer: URLayer[
-    AppConfig & ConstructResponseUtilV2 & IriService & KnoraProjectService & MessageRelay & PermissionUtilADM &
-      ResourceUtilV2 & StandoffTagUtilV2 & SearchResponderV2 & StringFormatter & TriplestoreService & OntologyRepo,
-    ResourcesResponderV2,
-  ] = ZLayer.fromZIO {
+  type Dependencies = AppConfig & ConstructResponseUtilV2 & IriService & KnoraProjectService & MessageRelay &
+    PermissionUtilADM & ResourceUtilV2 & StandoffTagUtilV2 & SearchResponderV2 & StringFormatter & TriplestoreService &
+    OntologyRepo & PermissionsResponder
+
+  val layer: URLayer[Dependencies, ResourcesResponderV2] = ZLayer.fromZIO {
```

Comment:
suggestion (non-blocking): how would you feel about simply leaving out the type annotation here? (provided it can infer them correctly and it would still work, of course - I haven't checked)